qubes-rpc/qubes.DesktopNotify

```diff
@@ -216,6 +216,10 @@ def parse_app_icon(vm, untrusted):
         if "/tmp/" in untrusted_path:
             return ""
 
+        real_path = os.path.realpath(untrusted_path)
```

Comment:
Not sure if depending on libc `realpath(3)` is a good idea.  @marmarek what do you think?  @3hhh Thanks for the fix!